src/components/Navigation.js

```diff
@@ -32,45 +32,41 @@ class Navigation extends Component {
   render() {
     const { classes } = this.props;
     return (
-      <div className="App">
-        <Router>
-          <Fragment>
-            <header>
-              <div>
-                <AppBar position="static">
-                  <Toolbar>
-                    <IconButton
-                      className={classes.menuButton}
-                      color="inherit"
-                      aria-label="Menu"
-                    >
-                      <MenuIcon />
-                    </IconButton>
-                    <Typography
-                      variant="title"
-                      color="inherit"
-                      className={classes.flex}
-                    >
-                      TMDB
-                    </Typography>
-                    <nav>
-                      <ul className="navigation-menu">
-                        {routes.map((route, idx) =>
-                          this.renderRouteLinkListItem(idx, route)
-                        )}
-                      </ul>
-                    </nav>
-                  </Toolbar>
-                </AppBar>
-              </div>
-            </header>
-            <Route exact path="/" component={MainPage} />
-            <Route exact path="/actor" component={ActorPage} />
-            <Route exact path="/movie" component={MoviePage} />
-            <Route exact path="/list" component={MoviesList} />
-          </Fragment>
-        </Router>
-      </div>
+      <Router>
+        <Fragment>

```

Comment:
I see this `Fragment` third time 🙄 Pls, can you remove it?

Reply:
But with out it I have a error "A <Router> may have only one child element"

Reply:
Ok, I see. Than this one is resolved